library/iFixit/Matryoshka/Memcached.php

```diff
@@ -91,9 +91,8 @@ public function getMultiple(array $keys) {
          $found = $this->memcached->getMulti(array_keys($keys),
           \Memcached::GET_PRESERVE_ORDER);
       } else {
-         $cas_tokens = null;
-         $found = $this->memcached->getMulti(array_keys($keys), $cas_tokens,
```

Comment:
Looks like argument 2 is *optional*: https://github.com/php-memcached-dev/php-memcached/blob/03f034f63ea8224cf0072f9bcf3fd9c7b6141290/php_memcached.c#L1563

I remember we did this so Matryoshka would work with multiple versions of the extension. SEe #18

Reply:
Neat! Kill it!

Reply:
https://github.com/iFixit/Matryoshka/pull/24/commits/5a8eec3f2b2b19717aa49cf7ff9ae307dad8ed27